backend/src/main/resources/db/migration/internal/V0.063__create_join_mission_controllers_tables.sql

```diff
@@ -0,0 +1,54 @@
+create table missions_control_units (
+    id serial primary key,
+    mission_id integer references missions not null,
+    control_unit_id integer references control_units(id) not null,
+    contact text,
+    unique (mission_id, control_unit_id)
+);
+
+create table missions_control_resources (
+    id serial primary key,
+    mission_id integer references missions not null,
+    control_resource_id integer references control_resources(id) not null,
+    unique (mission_id, control_resource_id)
+);
+
+-- Add control units
+WITH missions_units_name AS (
+    select
```

Comment:
```suggestion
    select distinct
```
Il n'y a pas de contrainte d'unicité.